src/verification/utils/slices/slices.gobra

```diff
@@ -0,0 +1,221 @@
+// Copyright 2022 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +gobra
+
+package slices
+
+// How to extend this file:
+// - if we need to support slices of non-supported types, we must repeat all definitions
+//   for that type. For this, we should be careful to avoid introducing cylical dependencies.
+//   The suffix of the predicate/function should be the type of the elems of the slices.
+// - For each type, there might be two different types of operations: those that keep track
+//   of contents (the name of the operation ends in "C"), and those who do not.
+
```

Comment:
Did you try this?

Reply:
I have added some additional functions and predicates to slices.gobra and definition.gobra (notably `ViewRunes` and the `Runes` predicate). I do believe that it may make sense to eventually add those upstream to `gobra-libs`. Should I go through with including `gobra-libs` and add a (temporary) file that contains my added functions?

Reply:
I think it would make a lot of sense to move this to a package named `runeslice` in gobra-libs. Please use the same naming conventions here as we use for `byteslice` (i.e., rename `ViewRunes` to `View`) and provide a body to `View` - should be the same exact one as `byteslice.View`